packages/patternfly-3/patternfly-react-extensions/src/components/BadgedResource/BadgedResource.js

```diff
@@ -99,7 +99,7 @@ BadgedResource.propTypes = {
   /** Full name of the resource kind to display in the badge tooltip and for screen readers (defaulted if valid resourceKind is given) */
   kindStr: PropTypes.string,
   /** Name of the resource */
-  resourceName: PropTypes.string,
+  resourceName: PropTypes.node,
```

Comment:
While type node will accept a string, we typically use PropTypes.oneOfType([PropTypes.string, PropTypes.node]) to make it more clear.